ddtrace/settings/config.py

```diff
@@ -286,7 +286,7 @@ def __init__(self):
                 log.warning("Invalid obfuscation pattern, disabling query string tracing")
                 self.http_tag_query_string = False  # Disable query string tagging if malformed obfuscation pattern
 
-        self._ci_visibility_agentless_enabled = asbool(os.getenv("DD_CIVISIBILITY_AGENTLESS_ENABLED", default=False))
+        self._ci_visibility_enabled = asbool(os.getenv("DD_CIVISIBILITY_ENABLED", default=False))
```

Comment:
Does `DD_CIVISIBILITY_ENABLED` match with other tracer client libraries? I'm confused what the purpose of this environment variable is.

Reply:
When I made this change, I forgot that this envvar is defined in the spec. I'll revert it.